gluten-core/src/main/java/org/apache/gluten/substrait/expression/BinaryLiteralNode.java

```diff
@@ -33,6 +36,14 @@ public BinaryLiteralNode(byte[] value, TypeNode typeNode) {
 
   @Override
   protected void updateLiteralBuilder(Builder literalBuilder, byte[] value) {
-    literalBuilder.setBinary(ByteString.copyFrom(value));
+    ByteString byteValue;
+    try {
+      Method m = ByteString.class.getDeclaredMethod("wrap", byte[].class);
+      m.setAccessible(true);
+      byteValue = (ByteString) m.invoke(null, value);
+    } catch (NoSuchMethodException | InvocationTargetException | IllegalAccessException e) {
+      throw new RuntimeException(e);
+    }
+    literalBuilder.setBinary(byteValue);
```

Comment:
shall we fallback to use the "copy" code path here?

Reply:
These exceptions only occur when the code is wrong, so we should not hide the exception